A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.html

```diff
@@ -13,7 +13,7 @@
             </g>
             <g id="sts-reminder">
                 <rect x="350" y="572" width="50" height="20" fill="black" stroke="white" stroke-width="2" />
-                <text x="375" y="588.5" fill="white" font-size="20px" text-anchor="middle">STS</text>
+                <text x="375" y="588.5" fill="white" style="font-size: 20px !important;" text-anchor="middle">STS</text>
```

Comment:
Some CSS must be overriding the font-size. It's better to figure out which CSS rule is causing this. No need to put `!important` here when you know.